.github/workflows/ci.yml

```diff
@@ -45,8 +45,14 @@ jobs:
           sudo cp unicode.mapping /etc/apache2/
           sudo mkdir -p /var/cache/modsecurity
           sudo chown -R www-data:www-data /var/cache/modsecurity
+      - name: first check config (to get syntax errors)
+        run: sudo apachectl configtest
       - name: start apache with module
-        run: |
-          sudo systemctl restart apache2.service
-          sudo cat /var/log/apache2/error.log
-
+        run: sudo systemctl restart apache2.service
+      - name: Show httpd error log
+        if: always()
+        run: sudo cat /var/log/apache2/error.log
```

Comment:
```suggestion
        run: |
          errors="$(grep "ERROR" /var/log/apache2/error.log)"
          if [ -n "${errors}" ]; then
            echo "Found errors in error.log"
            echo "${errors}"
            exit 1
          fi
```

Reply:
This is just a proposal, the `grep` match might need to be adjusted in order to fail the job when it shouldn't and to properly match errors. I do think that the job should fail whenever an error is encountered though, since that's not a normal situation and should be investigated.

Reply:
Very good idea to grep for ERROR and mark as failed if found (and maybe WARNING) in the future.
However, as follow-up of my previous comment, we should also have the complete log available.
PS: the syntax of error log status is ":error]" & ":warn]"

Reply:
> However, as follow-up of my previous comment, we should also have the complete log available.

Makes sense.

Reply:
> @marcstern - could you align the CI? Then we should merge this PR.

done

Reply:
This conversation is still unresolved.

Reply:
@marcstern - could you check this conversation?